src/lib.rs

```diff
@@ -420,6 +419,7 @@ impl Plugin for CosmicEditPlugin {
                 swash_cache: SwashCache::new(),
             })
             .insert_resource(CosmicFontSystem(font_system))
+            .insert_resource(ClickTimer(Timer::from_seconds(0.5, TimerMode::Once)))
```

Comment:
Is this for https://github.com/StaffEngineer/bevy_cosmic_edit/issues/45 ? :)

Reply:
Oh yeah forgot to swap that to another branch lol